de/html/README.md

```diff
@@ -203,14 +203,14 @@ Nachdem wir das gemacht haben, laden (push) wir unsere Änderungen auf GitHub:
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 
-    $ cd ~/<dein pythonanywere-Benutzername>.pythonanywhere.com
+    $ cd ~/<deine-pythonanywhere-domain>.pythonanywhere.com
     $ git pull
     [...]
     
 
-(Denke daran &lt;dein pythonanywere-Benutzername> durch deinen PythonAnywhere-Benutzernamen zu ersetzen - ohne spitze Klammern).
```

Comment:
Oh, here's where I copy-pasted the wrong spelling from, I guess. :stuck_out_tongue: